src/zimdump.cpp

```diff
@@ -416,7 +416,12 @@ int subcmdDump(ZimDumper &app,  std::map<std::string, docopt::value> &args)
     
     std::string directory = args["--dir"].asString();
 
-    if (*directory.rbegin() == '/'){
+    if (directory.empty()) {
+              throw std::runtime_error(
+                std::string("Directory cannot be empty."));
+    }
```

Comment:
1. Too much indentation
2. Write the throw statement on one line, and omit `std::string` (it is just extra noise when reading the code).